frontend/src/filters/amount.js

```diff
@@ -0,0 +1,18 @@
+let i18n
+
+export const loadFilters = (_i18n) => {
+  i18n = _i18n
+  return { amount, GDD }
+}
+
+const amount = (value) => {
+  if (!value && value !== 0) return ''
+  return i18n.n(value.toString(), 'decimal').replace('-', '− ')
+}
+
+const GDD = (value) => {
+  value = amount(value)
+  if (value === '') return ''
+  if (!value.match(/^− /)) value = '+ ' + value
+  return value + ' GDD'
+}
```

Comment:
why two functions. You seem to never call amount outside of the GDD function. Combining both could also save you the regex match.

Reply:
`amount` is used. I think of implement a `GDT` filter as well, so it is good to have a helper function.